sdk/cosmosdbforpostgresql/Azure.ResourceManager.CosmosDBForPostgreSql/src/autorest.md

```diff
@@ -50,24 +50,27 @@ rename-rules:
   Etag: ETag|etag
 
 rename-mapping:
-  NameAvailability: CosmosDBForPostgreSqlClusterNameAvailabilityResult
-  NameAvailability.nameAvailable: IsNameAvailable
-  NameAvailability.type: -|resource-type
+  Configuration.properties.requiresRestart: IsRestartRequired
   CheckNameAvailabilityResourceType: CosmosDBForPostgreSqlNameAvailabilityResourceType
   CheckNameAvailabilityResourceType.Microsoft.DBforPostgreSQL/serverGroupsv2: ServerGroupsV2
-  NameAvailabilityRequest: CosmosDBForPostgreSqlClusterNameAvailabilityContent
   Cluster.properties.enableShardsOnCoordinator: IsShardsOnCoordinatorEnabled
   Cluster.properties.enableHa: IsHAEnabled
   Cluster.properties.coordinatorEnablePublicIpAccess: IsCoordinatorPublicIPAccessEnabled
   Cluster.properties.nodeEnablePublicIpAccess: IsNodePublicIPAccessEnabled
-  ClusterServer.properties.enableHa: IsHAEnabled
-  ClusterServer.properties.enablePublicIpAccess: IsPublicIPAccessEnabled
-  Configuration.properties.requiresRestart: IsRestartRequired
-  ServerConfiguration.properties.requiresRestart: IsRestartRequired
   ClusterForUpdate.properties.enableShardsOnCoordinator: IsShardsOnCoordinatorEnabled
   ClusterForUpdate.properties.enableHa: IsHAEnabled
   ClusterForUpdate.properties.coordinatorEnablePublicIpAccess: IsCoordinatorPublicIPAccessEnabled
   ClusterForUpdate.properties.nodeEnablePublicIpAccess: IsNodePublicIPAccessEnabled
+  ClusterServer.properties.enableHa: IsHAEnabled
+  ClusterServer.properties.enablePublicIpAccess: IsPublicIPAccessEnabled
+  FirewallRule.properties.startIpAddress: -|ip-address
+  FirewallRule.properties.endIpAddress: -|ip-address
+  NameAvailability: CosmosDBForPostgreSqlClusterNameAvailabilityResult
+  NameAvailability.nameAvailable: IsNameAvailable
+  NameAvailability.type: -|resource-type
+  NameAvailabilityRequest: CosmosDBForPostgreSqlClusterNameAvailabilityContent
+  ProvisioningState: ClusterProvisioningState
```

Comment:
Shall we just prepend PR name to `ProvisioningState`? The type name is taken in avs and networkcloud already.